setup.py

```diff
@@ -354,7 +354,7 @@ def setup_package():
     NRN_COLLECT_DIRS = ["bin", "lib", "include", "share"]
 
     docs_require = []  # sphinx, themes, etc
-    maybe_rxd_reqs = ["numpy<2", "Cython<3"] if Components.RX3D else []
```

Comment:
(silly question, but IIRC, it seemed like `numpy<2` was required by the old Cython? Can we remove that pin, too?)

Reply:
I have to investigate if there are other issues with `numpy>=2`, but that was indeed the reason for prioritizing getting rid of the `cython<3` pin.